app/javascript/packs/base.jsx

```diff
@@ -4,10 +4,6 @@ import {
   getInstantClick,
   initializeMemberMenu,
 } from '../topNavigation/utilities';
-import * as Routes from '@routes';
```

Comment:
Reverting this for now because for some reason, locally this causes Cypress tests to timeout. This needs to be investigated further in a subsequent PR as it's currently a blocker. cc: @Ridhwana